src/__tests__/Raids.test.tsx

```diff
@@ -0,0 +1,131 @@
+/*eslint no-irregular-whitespace: ["error", { "skipTemplates": true }]*/
+import * as React from 'react'
+import { loadingScreen, render, screen, userEvent } from '../tests/testUtils'
+import App from '../App'
+import {
+  fetchedCollectionData,
+  fetchedDocumentData,
+} from '../tests/data/raidsData'
+
+// Note: After each Test Jest doesn't un-mount `The Component` Thats why we don't need to re-visit the page every time
+
+beforeAll(() => {
+  jest.doMock('../utils/useDocument', () =>
+    jest.fn().mockReturnValue(fetchedDocumentData),
+  )
+  jest.doMock('../utils/useCollection', () =>
+    jest.fn().mockReturnValue(fetchedCollectionData),
+  )
+})
+afterAll(() => {
+  jest.resetAllMocks()
+})
+const data = fetchedDocumentData.data
+
+it('should test the ViewRaid $TotalStats', async () => {
```

Comment:
Can we move checking this data to the integration test?

Also not sure if the filtering should be moved there as well...